python/src/HSICStatImplementation.i

```diff
@@ -0,0 +1,23 @@
+// SWIG file HSICStatImplementation.i
+
+// do not pass argument by reference, return it as tuple item
+//%typemap(in, numinputs=0) OT::Gamma & dist ($*ltype temp) %{ temp = OT::Gamma(); $1 = &temp; %}
+//%typemap(argout) OT::Gamma & dist %{ $result = SWIG_Python_AppendOutput( SWIG_NewPointerObj(new OT::Gamma(*$1), SWIGTYPE_p_OT__Gamma, SWIG_POINTER_OWN |  0 ), $result)  ; %}
+
+%{
+#include "openturns/HSICStatImplementation.hxx"
+%}
+
+%include Gamma.i
```

Comment:
this is not allowed, the hsic classes must be in a module higher than the distribution modules

Reply:
I do not get it. How can I use the Gamma here ?

Reply:
the hsic classes must be included in a swig module that have dist_bundle2_module.i: like bayesian (instead of simulation)

Reply:
By elimination I would put that in uncertainty_module.i. Is it good for you?

Reply:
that works too

Reply:
Done